package.json

```diff
@@ -57,7 +57,6 @@
     "start:development:storybook": "storybook dev --config-dir .storybook --no-open --no-version-updates --port 6006",
     "start:development:ts-morph": "chokidar ./custom-elements.json './src/ts-morph/**' --initial --silent --command 'tsx ./src/ts-morph/run.ts'",
     "start:development:ts-morph:comment:tsx": "Remove `tsx` once Node.js type stripping is available.",
-    "start:development:ts-morph:comment:watching": "It would be helpful to also watch './custom-elements.json`. But doing so would create a loop between this script and `start:development:cem-analyze`.",
```

Comment:
Cruft. I [managed](https://github.com/CrowdStrike/glide-core/blob/main/src/ts-morph/run.ts#L93-L95) to stop the loop.